static/layout.less

```diff
@@ -1,44 +1,44 @@
 #everything {
-  .flex;
-  -webkit-flex-flow: row wrap;
-  justify-content: flex-start;
-  align-items: stretch;
-  flex-wrap: nowrap;
   box-sizing: border-box;
-  height: calc(~"100% - 53px");// 53px header
   -webkit-overflow-scrolling: touch;
   padding-bottom: 0;
   transition: padding-bottom @transition-speed ease;
+  display:-ms-grid;
+  display: grid;
+  -ms-grid-columns: 300px 1fr;
+  grid-template-columns: max-content 1fr;
```

Comment:
I just realized `max-content` isn’t supported by Edge. Maybe we should also add a rule for `fit-content(50%)` or something else that would work? https://developer.mozilla.org/en-US/docs/Web/CSS/fit-content